pkg/securelaunch/policy/policy.go

```diff
@@ -33,148 +37,178 @@ type Policy struct {
 // policyBytes is a byte slice to hold a copy of the policy file in memory.
 var policyBytes []byte
 
-// scanKernelCmdLine scans the kernel cmdline for the 'sl_policy' flag.
-// When set it provides location of the policy file on disk. It reads it and
-// returns the policy file as a byte slice.
-//
-// The format of sl_policy flag is as follows:
-// sl_policy=<block device identifier>:<path>
-// e.g., sda:/boot/securelaunch.policy
-// e.g., 4qccd342-12zr-4e99-9ze7-1234cb1234c4:/foo/securelaunch.policy
-func scanKernelCmdLine() []byte {
-	slaunch.Debug("scanKernelCmdLine: scanning kernel cmd line for *sl_policy* flag")
-	val, ok := cmdline.Flag("sl_policy")
-	if !ok {
-		log.Printf("scanKernelCmdLine: sl_policy cmdline flag is not set")
-		return nil
+// pubkeyBytes is a byte slice to hold a copy of the public key file in memory.
+var pubkeyBytes []byte
+
+// signatureBytes is a byte slice to hold a copy of the signature file in memory.
+var signatureBytes []byte
+
+// Load reads the specified policy file. If pubkey and signature files are
+// also set, then load them as well. Note that the pubkey and signature files
+// need to be set together (i.e., either they are both set or neither is set).
+func Load(policyLocation, pubkeyLocation, signatureLocation string) error {
+	var err error
```

Comment:
I'm not sure if this is needed or I missing something?

Reply:
It is needed for now because the call to ```        policyBytes, err = slaunch.ReadFile(policyLocation)```  is modifying a global. 

That needs to get fixed later.